### PR TITLE
Add usage examples

### PR DESCRIPTION
**Pull request by theSoenke**

### CODE REVIEW

**Review by timleers — APPROVED** [review]
👍

File: README.md [hunk 1]
```diff
@@ -2,6 +2,109 @@
...
+keyName := "key_name"
+tags := "tag1, tag2"
+keyParams := phraseapp.TranslationKeyParams{
+	Name: &keyName,
+	Tags: &tags,
+}
+key, err := client.KeyCreate("project_id", &keyParams)
+```
+
+### Upload translation
```
Comment (sacry-dyn) [inline README.md]: Create

File: README.md [hunk 2]
```diff
@@ -2,6 +2,109 @@
...
+```
+keyID := "key_id"
+localeID := "locale_id"
+content := "my_content"
+translationParams := phraseapp.TranslationParams{
+    KeyID:    &keyDetails.ID,
```
Comment (sacry-dyn) [inline README.md]: Where is this keyDetails coming from?

**Review by sacry-dyn — APPROVED** [review]
Look very good, two minor points.

**Comment by theSoenke** [comment]
@sacry-dyn fixed

**Pull request merged**